src/main/java/com/jfb/ticket_app/Main.java

```diff
@@ -0,0 +1,24 @@
+package com.jfb.ticket_app;
+
+import com.jfb.ticket_app.model.user.Admin;
+import com.jfb.ticket_app.model.user.Client;
+import com.jfb.ticket_app.repository.dao.TicketDaoImpl;
+import com.jfb.ticket_app.repository.dao.UserDaoImpl;
+
+public class Main {
+
+  public static void main(String[] args) {
+    TicketDaoImpl ticketDao = new TicketDaoImpl();
+    UserDaoImpl userDao = new UserDaoImpl();
```

Comment:
if you create interface, then use it:
```java
GeneralDao<Ticket> ticketDao = new TicketDaoImpl();
GeneralDao<User> userDao = new UserDaoImpl();
```

Reply:
decided to get rid of the redundunt interface.